types.d.ts

```diff
@@ -3236,17 +3236,7 @@ declare abstract class ExportInfo {
 		| "maybe provided (runtime-defined)"
 		| "provided"
 		| "not provided";
-	getRenameInfo():
-		| string
-		| "missing provision and use info prevents renaming"
-		| "usage prevents renaming (no provision info)"
-		| "missing provision info prevents renaming"
-		| "missing usage info prevents renaming"
-		| "usage prevents renaming"
-		| "could be renamed"
-		| "provision prevents renaming (no use info)"
-		| "usage and provision prevents renaming"
-		| "provision prevents renaming";
+	getRenameInfo(): string;
```

Comment:
running the types generator caused this FYI. which seems right since it will have always ultimately been `string` before anyway but not sure why its turned up here (someone forgot to run it previously?)

Reply:
`yarn run special-lint-fix` produces different types to `npm run special-lint-fix`, fun one. these are now back after using yarn